config/src/main/java/io/scalecube/config/ConfigRegistryImpl.java

```diff
@@ -81,25 +84,31 @@ protected boolean removeEldestEntry(Map.Entry<ConfigEvent, Object> eldest) {
         }
       };
 
+
   ConfigRegistryImpl(ConfigRegistrySettings settings) {
     Objects.requireNonNull(settings, "ConfigRegistrySettings can't be null");
     this.settings = settings;
   }
 
-  void init() {
+  public void init() {
```

Comment:
why does it have the public modifier?

Reply:
**Codacy** told me to do that

Reply:
Codacy is evil - please remove this modifier.